packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -168,34 +168,13 @@ export class FASTButton extends FormAssociatedButton {
 
         this.proxy.setAttribute("type", this.type);
         this.handleUnsupportedDelegatesFocus();
-
-        const elements = Array.from(this.control?.children) as HTMLSpanElement[];
-        if (elements) {
-            elements.forEach((span: HTMLSpanElement) => {
-                span.addEventListener("click", this.handleClick);
-            });
-        }
-    }
-
-    /**
-     * @internal
-     */
-    public disconnectedCallback(): void {
-        super.disconnectedCallback();
-
-        const elements = Array.from(this.control?.children) as HTMLSpanElement[];
-        if (elements) {
-            elements.forEach((span: HTMLSpanElement) => {
-                span.removeEventListener("click", this.handleClick);
-            });
-        }
     }
 
     /**
      * Prevent events to propagate if disabled and has no slotted content wrapped in HTML elements
      * @internal
      */
-    private handleClick = (e: Event) => {
+    public handleClick = (e: Event) => {
         if (this.disabled && this.defaultSlottedContent?.length <= 1) {
             e.stopPropagation();
         }
```

Comment:
Is there a reason we need to keep this? If this is disabled it'll be marked as such on the internal button. I'd suggest removing this and removing the click handler on the button.

Reply:
Done